test/libsolidity/semanticTests/abiencodedecode/abi_encode_call.sol

```diff
@@ -0,0 +1,31 @@
+pragma abicoder v2;
+contract C {
+	function f(uint256 p) public {}
+
+	function f0() public pure returns (bytes memory) {
+		return abi.encodeWithSignature("f(uint256)", 1);
```

Comment:
The test expectation below is unreadable. Can you add a function `test()` that checks if the keccak hashes of the two byte arrays are equal?